retrieve.py

```diff
@@ -3,15 +3,18 @@
 import re
 from bs4 import BeautifulSoup
 from geopy import geocoders
+from datetime import datetime
 
 header = {
     # secret
     }
 
-from datetime import datetime
+
+#Getting the current date in YY:MM:DD to set as default day
 today = datetime.today()
 formatted_date = today.strftime("%Y %m %d")
 
+#Cuisines on Resy Website
 applicable_cuisine_list = [
```

Comment:
is this it? aren't we letting user choose these?

Reply:
These are the only ones on resy we can query for